tests/unit/utils/test_utils_record.py

```diff
@@ -52,28 +54,145 @@ def single_title():
 
 
 @pytest.fixture
+def title_only_from_arxiv():
+    return {
+        "titles": [
+            {
+                "source": "arXiv",
+                "title": "Title from arxiv",
+                "subtitle": "Subtitle from arxiv"
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def double_title_with_multiple_sources():
+    return {
+        "titles": [
+            {
+                "source": "arXiv",
+                "title": "Parton distributions with LHC data",
+                "subtitle": "Parton distributions with LHC data"
+            },
+            {
+                "source": "other_source",
+                "title": "Title of other_source",
+                "subtitle": "Subtitle of other_source"
+            }
+        ]
+    }
+
+
+@pytest.fixture
 def empty_title():
     return {
         "titles": []
     }
 
 
-def test_get_title(double_title, single_title, empty_title):
+@pytest.fixture
+def double_abstract():
+    return {
+        "abstracts": [
+            {
+                "source": "arXiv",
+                "value": "arXiv abstract"
+            },
+            {
+                "value": "abstract"
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def abstract_only_from_arxiv():
+    return {
+        "abstracts": [
+            {
+                "source": "arXiv",
+                "value": "arXiv abstract"
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def double_abstract_with_multiple_sources():
+    return {
+        "abstracts": [
+            {
+                "source": "arXiv",
+                "value": "abstract"
+            },
+            {
+                "source": "other_source",
+                "value": "other_source abstract",
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def single_abstract():
+    return {
+        "abstracts": [
+            {
+                "value": "abstract",
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def empty_abstract():
+    return {
+        "abstracts": []
+    }
+
+
+def test_get_title(double_title, single_title, empty_title, double_title_with_multiple_sources, title_only_from_arxiv):
```

Comment:
Some time ago I rewrote these tests in a style I like more, but I never pushed it anywhere: https://github.com/jacquerie/inspire-next/commit/ecb6826a4fa169efea634dc3672cc59edc4c4d09